src/build.d

```diff
@@ -270,6 +293,15 @@ auto dBackend()
 auto cxxBackend()
 {
     Dependency[] dependencys;
```

Comment:
`dependencies` - sorry for the typo :O

Reply:
I will fix that later.  There are a few places it needs to change and I'd like to keep the diff low.